subprojects/platform-play/src/testFixtures/groovy/org/gradle/play/integtest/fixtures/PlayMultiVersionApplicationIntegrationTest.groovy

```diff
@@ -25,6 +26,7 @@ abstract class PlayMultiVersionApplicationIntegrationTest extends PlayMultiVersi
     abstract PlayApp getPlayApp()
 
     def setup() {
+        playApp.writeSources(testDirectory)
```

Comment:
A possible improvement might be to build the version into `PlayApp` somehow so that this ordering is taken care of in `writeSources`.  We already do some massaging of the build file when writing sources - this could just be a part of that.

Another thing we might include is some verification that we are getting the version we want.  For instance, we might use a flag to tell gradle to print the play version and then we validate it as part of `succeeds()` below.